CODEOWNERS

```diff
@@ -63,8 +63,8 @@
 /i18n/ru.toml @shurup @kirkonru @tym83
 
 # Approvers for Turkish contents
-/content/tr/ @aliok @halil-bugol @rwxdash @eminalemdar
-/i18n/tr.toml @aliok @halil-bugol @rwxdash @eminalemdar
+/content/tr/ @aliok @symys @rwxdash @eminalemdar
+/i18n/tr.toml @aliok @symys @rwxdash @eminalemdar
```

Comment:
There is a warning about write access here, we might need someone to update the permissions for `@symys`.

Reply:
I think its ok, I often saw this and it should be solved as soon as we have it merged.